src/app/pages/LandingPage.svelte

```diff
@@ -6,21 +6,31 @@
   import DownloadKMP from "../components/DownloadKMP.svelte";
   import SplitButton from "../components/SplitButton.svelte";
   import { currentDownloadURL } from "../stores";
+  import type { KeyboardMetadata } from "@common/types";
+  import type { RelevantKmpOptions } from "@common/kmp-json-file";
 
-  let languageStatus: boolean = false;
+  let selectedLanguage: KeyboardMetadata | undefined = undefined;
   let continueReady: boolean = false;
   let uploadFile: boolean = true;
 
-  $: continueReady = languageStatus && Boolean($currentDownloadURL);
+  $: continueReady =
+    selectedLanguage !== undefined && Boolean($currentDownloadURL);
```

Comment:
```suggestion
    selectedLanguage && Boolean($currentDownloadURL);
```

Reply:
Unfortunately, Typescript complains when I do this due to the comparison with a non-boolean and a boolean. 

I could go `Boolean(selectedLanguage) && Boolean($currentDownloadURL)`, but personally I like being explicit with my types. I feel that casting `selectedLanguage` to a Boolean goes against the explicit nature of typing `selectedLanguage` in the first place.

What are your thoughts?